### PR TITLE
Adding new test to check no failed events actions

### PR DESCRIPTION
**Pull request by MalloZup**
This test will check that no failed actions exist.
We have a layer of better debugging and regression testing within.

### CODE REVIEW

**Comment by MalloZup** [comment]
Testing it right now

File: features/spacewalk_debug.feature [hunk 1]
```diff
@@ -1,8 +1,7 @@
-# Copyright (c) 2015-16 SUSE LLC
+# Copyright (c) 2015-17 SUSE LLC
 # Licensed under the terms of the MIT license.
 
-Feature:  Call spacewalk-debug on the server
- 	  Check some generics logs 
+Feature:  Debug Suse-manager server after run testsuite
```
Comment (Bischoff) [inline features/spacewalk_debug.feature]: "Debug SUSE Manager after the testsuite has run"

File: features/step_definitions/xmlrpc_schedule_steps.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# COPYRIGHT 2017 SUSE LLC
+require "xmlrpc/client"
+
+scdrpc = XMLRPCScheduleTest.new(ENV['TESTHOST'])
+scdrpc.login('admin', 'admin')
+
+Then(/^there should be no failed scheduled actions$/) do
+  assert_empty(scdrpc.listFailedActions)
+end
```
Comment (Bischoff) [inline features/step_definitions/xmlrpc_schedule_steps.rb]: Beautiful.

**Comment by MalloZup** [comment]
ok thx for review, merging it

**Pull request merged**